doc/releases/changelog-0.28.0.md

```diff
@@ -771,67 +624,60 @@ Deprecations cycles are tracked at [doc/developement/deprecations.rst](https://d
+  [(#3505)](https://github.com/PennyLaneAI/pennylane/pull/3505)
 
-<h3>Bug fixes</h3>
+<h3>Bug fixes 🐛</h3>
 
 * Fixed a bug where `qml.transforms.hamiltonian_expand` didn't preserve the type of the input results in its output.
   [(#3339)](https://github.com/PennyLaneAI/pennylane/pull/3339)
 
 * Fixed a bug that made `qml.gradients.param_shift` raise an error when used with unshifted terms only in a custom recipe, and when using any unshifted terms at all under the new return type system.
   [(#3177)](https://github.com/PennyLaneAI/pennylane/pull/3177)
 
-* Original tape `_obs_sharing_wires` attribute is updated during its expansion.
+* The original tape `_obs_sharing_wires` attribute is updated during its expansion.
   [(#3293)](https://github.com/PennyLaneAI/pennylane/pull/3293)
 
-* An issue with `drain=False` in the adaptive optimizer has been fixed. Before the fix, the operator pool needed to be reconstructed inside the optimization pool when `drain=False`. With this improvement, this reconstruction is no longer needed.
+* An issue with `drain=False` in the adaptive optimizer has been fixed. Before the fix, the operator pool needed to be reconstructed inside the optimization pool when `drain=False`. With this fix, this reconstruction is no longer needed.
   [(#3361)](https://github.com/PennyLaneAI/pennylane/pull/3361)
 
-* If the device originally has no shots but finite shots are dynamically specified, Hamiltonian
-  expansion now occurs.
+* If the device originally has no shots but finite shots are dynamically specified, Hamiltonian expansion now occurs.
   [(#3369)](https://github.com/PennyLaneAI/pennylane/pull/3369)
 
-* `qml.matrix(op)` now fails if the operator truly has no matrix (e.g., `qml.Barrier`) to match `op.matrix()`
+* `qml.matrix(op)` now fails if the operator truly has no matrix (e.g., `qml.Barrier`) to match `op.matrix()`.
   [(#3386)](https://github.com/PennyLaneAI/pennylane/pull/3386)
 
-* The `pad_with` argument in the `qml.AmplitudeEmbedding` template is now compatible with all interfaces
+* The `pad_with` argument in the `qml.AmplitudeEmbedding` template is now compatible with all interfaces.
   [(#3392)](https://github.com/PennyLaneAI/pennylane/pull/3392)
 
 * `Operator.pow` now queues its constituents by default.
   [(#3373)](https://github.com/PennyLaneAI/pennylane/pull/3373)
 
-* Fixed a bug where a QNode returning `qml.sample` would produce incorrect results when
-  run on a device defined with a shot vector.
+* Fixed a bug where a QNode returning `qml.sample` would produce incorrect results when run on a device defined with a shot vector.
   [(#3422)](https://github.com/PennyLaneAI/pennylane/pull/3422)
 
 * The `qml.data` module now works as expected on Windows.
   [(#3504)](https://github.com/PennyLaneAI/pennylane/pull/3504)
 
-<h3>Contributors</h3>
+<h3>Contributors ✍️</h3>
 
 This release contains contributions from (in alphabetical order):
 
```

Comment:
![image](https://user-images.githubusercontent.com/49409390/207692357-b18a91d4-0375-4252-bce8-24f638ac9e83.png)
This doesn't show up great (no commas). I think it's easier for devs to go 1-person-per-line, but when we finalize the changelog we should make sure it's comma separated.

Reply:
Sure thing!